Tests/TuistLoaderTests/SwiftPackageManager/SwiftPackageManagerModuleMapGeneratorTests.swift

```diff
@@ -68,9 +68,18 @@ class SwiftPackageManagerModuleMapGeneratorTests: TuistTestCase {
             writeCalled = true
             let expectedContent: String
             switch moduleMap {
-            case .none, .custom, .header:
+            case .none, .custom:
                 XCTFail("FileHandler.write should not be called")
                 return
+            case .header:
+                expectedContent = """
+                framework module Module {
+                  umbrella header "/Absolute/Public/Headers/Path/Module.h"
+
+                  export *
+                  module * { export * }
+                }
+                """
```

Comment:
I remember trying this road back when I have implemented the whole thing, and having the absolute path here is breaking remote caching, isn't it?

Reply:
Remote caching looks to be fine as the modulemaps need to be passed to the actual source targets – which they are even when cached. The modulemaps in xcframeworks are not these generated modulemaps but are different artifacts created by `xcodebuild` afaict.